dts/bindings/power-domain/power-domain-gpio-monitor.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2023 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Simple monitorig power domain
+
+  This power domain monitors the state of a GPIO pin to detect
+  whether a power rail is on/off. Due to asynchronous nature of
+  monitoring a pending transaction won't be interruped by power
+  state change.
```

Comment:
You have to add here that trying to suspend or resume it won't trigger any physical action and consequently won't trigger any action in children devices.